src/sources/postgresql_metrics.rs

```diff
@@ -535,7 +535,11 @@ impl PostgresqlMetrics {
             Ok(result) => {
                 let (count, byte_size, received_byte_size) =
                     result.iter().fold((0, 0, 0), |res, (set, size)| {
-                        (res.0 + set.len(), res.1 + set.size_of(), res.2 + size)
+                        (
+                            res.0 + set.estimated_json_encoded_size_of(),
+                            res.1 + set.size_of(),
+                            res.2 + size,
+                        )
```

Comment:
Is this right? It looks like `count` is being assigned the total of the estimated JSON encoded size, not of the set lengths.

Reply:
Ah crap no, I misread the ordering here, will fix.